debian/billboard.aegis

```diff
@@ -1,4 +1,17 @@
 <aegis>
+    <provide>
+        <credential name="control" />
+        <credential name="access" />
+        <dbus name="io.thp.billboard" own="control" bus="system">
```

Comment:
Above you say `QDBusConnection::sessionBus().registerService(DBUS_SERVICE);`, but you say `bus="system"` here -- I guess it should be the session bus?